packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -96,6 +97,10 @@ struct CreateEntityRequest {
     owned_by_id: OwnedById,
     entity_id: Option<EntityId>,
     actor_id: CreatedById,
+    // TODO: this could break invariants if we don't move to fractional indexing
```

Comment:
I think we should just check the final diff before merging `dev/links` in and ensure we don't have any languishing TODOs. **_If_** we agree to do that then I think leaving things like this for now is less risky.